process/block/shardblock.go

```diff
@@ -346,6 +347,11 @@ func (sp *shardProcessor) ProcessBlock(
 		return err
 	}
 
+	err = sp.blockProcessingCutoffHandler.HandleProcessErrorCutoff(header)
```

Comment:
return directly?

Reply:
done

Reply:
Wrong, we should not return directly. `err` variable is evaluated in the defer anonymous function.